watson/watson.py

```diff
@@ -247,6 +248,13 @@ def start(self, project, tags=None):
                 )
             )
 
+        default_tags = self._default_tags(project)
+        if default_tags and not restart:
+            if tags is None:
+                tags = default_tags
+            else:
+                tags = tags + default_tags
```

Comment:
I would rather keep the order, which is not the case with a set. There are ordered sets in python, but not in the standard library. The simplest way to keep the order without adding a dependancy would be to write a function to deduplicate the list, which should be rather easy.

Note that the change would broader the scope of the PR. Indeed, it would deduplicate tags provided multiple time by the user in cases such as:

```
watson start project +tag + tag
```

This would be an enhancement, I think, but I prefer to warn.